index.js

```diff
@@ -16,7 +16,7 @@ const findHight = () => {
   }
 };
 
-export default function loadBooks(bookCollection) {
+export default  function loadBooks(bookCollection) {
   sectionBooks.innerHTML = '';
   /* eslint-disable */
```

Comment:
- [ ] [OPTIONAL] Kindly disable the linters because the help you write clean and maintainable code that is up to standards.